2-mandatory/1-writers.js

```diff
@@ -72,6 +75,13 @@ Exercise 2:
 
 function logDeadWritersInTheirForties() {
   // write your code here
+  writers
+    .filter((writer) => writer.age >= 40 && writer.age < 50 && !writer.alive)
+    .forEach((writer) =>
+      console.log(
+        `Writer ${writer.firstName} ${writer.lastName} died at ${writer.age} years old.`
+      )
+    );
```

Comment:
Performance suggestion. Since the logical checks start from left to right. You may consider to check alive first to save potential checking on ages.